docs/experimental-features/security-posture-management.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[security-posture-management]]
+== Cloud security posture management
+
+NOTE: This feature requires Elastic Stack version 8.3.0 or later.
+
+Cloud security posture management (CSPM) and Kubernetes security posture management (KSPM) continuously monitor and compare your cloud and Kubernetes infrastructure against security best practices to help you identify and remediate misconfigurations.
+
+[discrete]
+=== Enable security posture management
+
+The steps to enable this feature differ between Elastic cloud and self-hosted deployments.
+
+[[enable-kspm-on-cloud]]
+[discrete]
+==== Enable KSPM for cloud deployments
+
+First, enable the KSPM flag for your deployment:
+
+* From {kib}, open the main menu and click on **Manage this deployment**.
+
+image::images/kspm-1.png[The Manage deployment button]
+
+1. Under `My deployment`, select **Kibana**.
+2. Under "Instances", click the three-dot menu next to your instance, then click **Edit configuration**.
+3. Click **Edit user settings** in the upper right of the page.
+4. Add `xpack.cloudSecurityPosture.enabled: true` to the User settings.
+5. Click **Back** then save your Kibana user settings.
+
+NOTE: It may take up to two minutes for the changes to take effect.
+
+Navigate back to Kibana. Open the main menu, then go to the Security section and click **Cloud Security**.
+
+image::images/kspm-2.png[The Cloud Security button on the main menu, width=160]
+
+Follow the prompts to set up data ingestion.
+
+[[enable-kspm-on-selfhosted]]
+[discrete]
+==== Enable KSPM for self-hosted deployments
```

Comment:
Similar thing here with numbered steps:
1. Edit the kibana.yml...
2. Navigate back...
3. Follow the prompts...